src/app/js/admin/progress/reducer.js

```diff
@@ -36,13 +38,29 @@ export default handleActions(
             symbol,
             error: undefined,
         }),
-        [combineActions(PUBLISH, UPLOAD_FILE)]: state => ({
+        [UPLOAD_FILE]: state => ({
             ...state,
             status: STARTING,
             progress: undefined,
             target: undefined,
             error: undefined,
         }),
+        [PUBLISH]: state => ({
```

Comment:
maybe [combineActions(PUBLISH, CLEAR_DATASET)] ?

Reply:
It was a mistake, CLEAR_DATASET shouldn't be background